nav_msgs/Path init_path
std_msgs/Float32[] max_vel_percentage
---
nav_msgs/Path generated_path
nav_msgs/Path generated_path_vel_percentage
std_msgs/Float32 max_velocity
std_msgs/Float32[] generated_max_vel_percentage